Makefile

```diff
@@ -79,7 +81,8 @@ TARGET = $(PROJECT_TARGET)
 DEBUG = 1
 # optimization
 OPT = -Og
-
+# verbose
+VERBOSE ?= 0
```

Comment:
> Also added a verbosity flag just because. Enabled by default.

Isn't this supposed to be 1 to be enabled by default?

Reply:
I figure we want less verbosity by default, as the usefulness in showing the entire compile command comes when debugging compilation specifically. Otherwise just a printed out "CC file.c -> file.o" will suffice